examples/api-routes-apollo-server-and-client/next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  publicRuntimeConfig: {
```

Comment:
Use [Build time-configuration](https://github.com/zeit/next.js#build-time-configuration) instead of `publicRuntimeConfig`.